### PR TITLE
feat(plugin-seo): add vietnamese translation

### PR DESCRIPTION
**Pull request by trangcongthanh**
## Description
- Fixed missing translation for the "Missing" string
- Added Vietnamese translation and updated index.ts

 I have read and understand the [CONTRIBUTING.md](https://github.com/payloadcms/payload/blob/main/CONTRIBUTING.md) document in this repository.
## Type of change
- [ ]  Chore (non-breaking change which does not add functionality)
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [x]  New feature (non-breaking change which adds functionality)
- [ ]  Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ]  Change to the [templates](https://github.com/payloadcms/payload/tree/main/templates) directory (does not affect core functionality)
- [ ]  Change to the [examples](https://github.com/payloadcms/payload/tree/main/examples) directory (does not affect core functionality)
- [ ]  This change requires a documentation update
## Checklist:
- [ ]  I have added tests that prove my fix is effective or that my feature works
- [ ]  Existing test suite passes locally with my changes
- [ ]  I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by jacobsfletch** [comment]
Thank you for the contribution @trangcongthanh! #8206 effects this PR a bit. Would you be willing to resolve the merge conflicts here?

**Comment by trangcongthanh** [comment]
@jacobsfletch I messed up my branch while sync with upstream. Do I need to create new PR?

**Comment by jacobsfletch** [comment]
I've reopened it for you.

File: packages/plugin-seo/src/ui/LengthIndicator.tsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ export const LengthIndicator: React.FC<{
     const textLength = text?.length || 0
 
     if (textLength === 0) {
-      setLabel('Missing')
+      setLabel(t('plugin-seo:missing'))
```
Comment (trangcongthanh) [inline packages/plugin-seo/src/ui/LengthIndicator.tsx:30]: @jacobsfletch I also want to translate the "Missing" string. If I introduce this i18n key, what should I do for other languages?

Comment (jacobsfletch) [reply packages/plugin-seo/src/ui/LengthIndicator.tsx:30]: You can either add translation strings for those languages, or they can remain in English as a fallback until translations have been added. If you add a new string, however, it is preferred that you also retrofit all other languages.

Comment (trangcongthanh) [reply packages/plugin-seo/src/ui/LengthIndicator.tsx:30]: I'm not using those languages so translate them may not fit with the context. But I used AI (Claude and ChatGPT 4o) to help add translation strings for those languages.

**Review by jacobsfletch — APPROVED** [review]

**Pull request merged**